mythril/laser/ethereum/state/environment.py

```diff
@@ -49,6 +50,7 @@ def __init__(
         self.gasprice = gasprice
         self.origin = origin
         self.callvalue = callvalue
+        self.static = static
```

Comment:
Variable name not clear.
How about `write_permission`

Reply:
Variable name is good right? Like it says something like `environment is static` if static is true,

Reply:
Its just that in the yellow paper this variable is called $$I_w$$ and the description is "the permission to make modifications to the state". 
The way to get this value to be true is "STATICCALL", which makes the name `static` a reasonable one. 
However, I feel like write_permission or has_write_permission directly confers what it is that the variable describes. 
Anyway, this is not really a dealbreaker 😅

Reply:
Lol nvm, documentation as always is not consistent. In other places they call the flag a static flag, so we can reasonably assume that anybody should be able to understand the variable name